apps/api/src/migration/1676405526677-CreateReportView.ts

```diff
@@ -1,444 +0,0 @@
-import _ from 'lodash';
-import { MigrationInterface, QueryRunner } from 'typeorm';
-import { STATUS } from '@ien/common';
```

Comment:
Whats the purpose of removing this migration?

Reply:
I thought we'd better remove STATUS.REFERRED_TO_NCAS to avoid confusion and this migration references it a lot. So I thought it's better to remove it cause it doesn't affect the sequence of migrations.